core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender43.java

```diff
@@ -60,6 +60,7 @@ public void run() {
                 PoolingHttpClientConnectionManager cm = new PoolingHttpClientConnectionManager();
                 cm.setMaxTotal(DEFAULT_MAX_TOTAL_CONNECTIONS);
                 cm.setDefaultMaxPerRoute(DEFAULT_MAX_CONNECTIONS_PER_ROUTE);
+                cm.setValidateAfterInactivity(REQUEST_TIMEOUT_IN_MILLIS);
```

Comment:
We should consider a smaller value for this.

Reply:
Based on what was mentioned in Stackoverflow article, the value for this should be 
`min(socketTime,  keep-alive time, server connection timeout, idleConnectionMonitor)` . The connections on backend as far as I know are always keepAlive. The server doesn't respond with this value. Neither I am aware of other two. Therefore, let's keep the value similar to SocketTimeOut for now. We will iterate if we find newer details